po/es.po

```diff
@@ -410,7 +371,7 @@ msgstr "configure una nueva habilidad"
 #~ msgstr "Elija un comando."
 
 #~ msgid "Do you want to use your own account or others?"
-#~ msgstr "¿Quiere usar su cuenta u otras?"
+#~ msgstr "¿Quiere usar su cuenta o otras?"
```

Comment:
Stuff at the end commented out is not used (it's left over from previous translations), you don't need to edit it.